packages/backend/src/database/seeds/development/02_saved_queries.ts

```diff
@@ -9,6 +9,7 @@ export async function seed(knex: Knex): Promise<void> {
     // Inserts seed entries
     await savedChartModel.create(SEED_PROJECT.project_uuid, {
```

Comment:
Since customers will see this on demo.lightdash.com, I'd like to make the descriptions more meaningful